File: PRAK604_2210817110001_AkhmadRaihanRidha.c
  #include <stdio.h>
#include <string.h>
int main(){
    char shikamaru1[20],shikamaru2[20];
    int panjang1,panjang2,i,A,B;
    A=0;
    B=0;

    scanf(" %[^\n]%*c", &shikamaru1);
    scanf(" %[^\n]%*c", &shikamaru2);

    panjang1=strlen(shikamaru1);

    panjang2=strlen(shikamaru2);

    if(panjang1!=panjang2)
    {
        printf("Panjang kalimat berbeda, pesan palsu");
        
    }
    else
    {
        for ( i = 0; i < panjang1; i++)
        {
            if (shikamaru1[i] == ' ' && shikamaru2[i] == ' ')
            {
                continue;
            }




            else if(shikamaru1[i] == shikamaru2[i])
            {
                printf("*");
                A++;
                
            }
            else
            {
                printf("#");
                B++;
            
            }
            
        }
        printf("\n* = %d",A);
        printf("\n# = %d\n",B);
        if (A >= B)
        {
            printf("Pesan Asli");
        }
        else
        {
            printf("Pesan Palsu");
        }
    }
    

File: PRAK604_2210817110001_AkhmadRaihanRidha.py
A=0
B=0
Char1=list(input(""))
Char2=list(input(""))
long1= len(Char1)
long2= len(Char2)

if long1 != long2:
    print("Panjang kalimat berbeda,pesan palsu")
else:
    for i in range (long1):
        if " "in Char1[i] and " "in Char2[i]:
            continue    
        elif Char1[i] == Char2[i]:
            print("*",end="")
            A += 1

        else:
            print("#", end="")
            B += 1
    print("\n* = ", A)
    print("# = ",B)
if A >= B:
    print("Pesan Asli")
else:
    print("Pesan Palsu")                